log/experimental_level/level.go

```diff
@@ -147,10 +63,87 @@ func (l *logger) Log(keyvals ...interface{}) error {
 	return l.next.Log(keyvals...)
 }
 
-func makeSet(a []string) map[string]struct{} {
-	m := make(map[string]struct{}, len(a))
-	for _, s := range a {
-		m[s] = struct{}{}
-	}
-	return m
+// Option sets a parameter for the leveled logger.
+type Option func(*logger)
+
+// AllowAll is an alias for AllowDebugAndAbove.
+func AllowAll() Option {
+	return AllowDebugAndAbove()
+}
+
+// AllowDebugAndAbove allows all of the four default log levels.
+func AllowDebugAndAbove() Option {
+	return allowed(levelDebug | levelInfo | levelWarn | levelError)
+}
```

Comment:
@peterbourgon, that's the inverse of my earlier commit in #453 (["Use a mask to filter log records by level"](https://github.com/go-kit/kit/pull/453/commits/134206ee8fc5b2dec23f2224d206f4b5c4cc6d07)). I started with the inequality that you propose here, but found that the bitmask implementation beat it in my benchmarks. Unfortunately, I didn't include the improvement in the commit message, but I recall it being on the order of 3%. It's worth testing again.